package.json

```diff
@@ -11,11 +11,12 @@
   "license": "MIT",
   "repository": "koajs/trie-router",
   "dependencies": {
-    "methods": "1",
+    "errors": "^0.3.0",
```

Comment:
Because we're creating a custom error with a specific code, not an HTTP error, right? I thought the intent was a custom error that the developer can handle and re-throw as another error like say a 404. Thoughs, @jonathanong?